plotting/matRad_plotPlan3D.m

```diff
@@ -109,7 +109,7 @@ function matRad_plotPlan3D(axesHandle,pln,stf)
         beamTarget = stf(fieldIx).isoCenter;
         beamSource = stf(fieldIx).sourcePoint + stf(fieldIx).isoCenter;
         
-        rotMat = matRad_getRotationMatrix(pln.gantryAngles(fieldIx),pln.couchAngles(fieldIx));
+        rotMat = matRad_getRotationMatrix(pln.propStf.gantryAngles(fieldIx),pln.propStf.collimatorAngles(fieldIx),pln.propStf.couchAngles(fieldIx));
```

Comment:
Here the angle is correctly assigned. If you change to third argument, please consider you need to adapt this one.